pkg/gen/node-templates/provider.ts.mustache

```diff
@@ -369,6 +368,17 @@ export namespace {{Group}} {
 
       {{/Properties}}
 
+      /**
+       * Get an existing {{Kind}} resource's state with the given name, ID, and optional extra
+       * properties used to qualify the lookup.
```

Comment:
I think this comment is not correct given that the extra properties are not exposed (and not needed)?